src/app/onion/learning-object-builder/components/scaffold/scaffold.component.ts

```diff
@@ -145,20 +145,23 @@ export class ScaffoldComponent implements OnInit {
   /**
    * Sends request to update the children array of the Learning Object
    */
-  deleteChild() {
+  async deleteChild() {
     this.toggleConfirmationModal(false);
     // remove the child that was selected to be deleted
     this.children.splice(this.deleteIndex, 1);
 
     // set childrenIDs equal to the children array
     this.childrenIDs = [];
     this.children.forEach(kid => this.childrenIDs.push(kid._id));
-    this.store.setChildren(this.childrenIDs, true);
+    await this.store.setChildren(this.childrenIDs, true);
```

Comment:
Huh, good find. Been like this for 7 years 😬

Reply:
yikes lol

Reply:
😮